ionoscloud/resource_server.go

```diff
@@ -1529,23 +1542,43 @@ func setResourceServerData(ctx context.Context, client *ionoscloud.APIClient, d
 		return fmt.Errorf("server entities cannot be empty for %s", d.Id())
 	}
 
-	if server.Properties != nil && server.Properties.BootVolume != nil {
-		volume, apiResponse, err := client.ServersApi.DatacentersServersVolumesFindById(ctx, datacenterId, d.Id(), *server.Properties.BootVolume.Id).Execute()
-		logApiRequestTime(apiResponse)
-		if err == nil {
-			var volumes []interface{}
+	inlineVolumeIds := d.Get("inline_volume_ids")
+
+	if inlineVolumeIds != nil {
+		inlineVolumeIds := inlineVolumeIds.([]interface{})
```

Comment:
very minor, but you can use any instead of interface{}

Reply:
currently there is mixed usage of `interface{}` and `any`, should we allow this or enforce one over the other?

Reply:
we can allow for now. any was added in go 1.18. maybe we do a search and replace at some point.